src/__tests__/userEvent/userEvent.press.test.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import { Pressable, Text, View } from 'react-native';
+import { render, screen, userEvent } from '../../pure';
+
+jest.useFakeTimers();
```

Comment:
If feasible it would be good to have test that also check real times behavior.

Reply:
I'm not in favor of supporting real timers for this for the reasons I explained in https://github.com/callstack/react-native-testing-library/pull/1386#discussion_r1172362950

Reply:
I agree that we should encourage users to use this library with fake timers, but I think we should still support using real times as some users might be reluctant to switch to fake timers, even if it would make their code run faster. Especially not all tests need to trigger `longPress` events.

Reply:
Indeed I think you're right. It should be doable to support both. In that case we should think of an appropriate way to recommend using fake timers. I'm thinking of a warning linking to a documentation, wdyt?

Reply:
I think we might add warning if they use some long duration action that affects the performance of the script, like the 500ms or more longer touch. In case of brief actions like regular press we should not bother them